common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -115,4 +115,9 @@ export default class AccountInfo extends React.Component<Props, State> {
       </div>
     );
   }
+  private getEthereumValue = (value: Wei, etherUnit: UnitKey) => (
+    formatNumber?: boolean
+  ) => (
+    <UnitDisplay value={value} unit={etherUnit} formatNumber={formatNumber} />
+  );
```

Comment:
This seems like it would be simpler if you just bassed `showLongBalance` as the third arg rather than returned a function that you then pass it into.

Reply:
Error was uncaught before because the function parameter was untyped, maybe we should look into setting `--strict` or at least `--noImplictAny` sometime soon

Reply:
Should be fixed now